comp/core/workloadmeta/collectors/internal/kubeapiserver/deployments.go

```diff
@@ -35,8 +35,11 @@ func (f *deploymentFilter) filteredOut(entity workloadmeta.Entity) bool {
 		(deployment.Env == "" &&
 			deployment.Version == "" &&
 			deployment.Service == "" &&
-			len(deployment.InitContainerLanguages) == 0 &&
-			len(deployment.ContainerLanguages) == 0)
+			len(deployment.InjectableLanguages.InitContainerLanguages) == 0 &&
+			len(deployment.InjectableLanguages.ContainerLanguages) == 0 &&
+			len(deployment.DetectedLanguages.InitContainerLanguages) == 0 &&
```

Comment:
It should always be empty (not populated by Parse)

Reply:
I was trying to cover the (rare) case where language detection is faster than deployment informer, which may result in a deployment entity that has only `deployment.DetectedLanguages` populated, and which should not be filtered out.

But I see that the `filteredOut` function is applied on the results of parse, so yes I agree this should always be empty 👍